howto/README.md

```diff
@@ -41,3 +41,7 @@ Here you'll find a list of How To guides that walk you through accomplishing spe
 ## Configuring Visual Studio Code
 
 * [Debugging with daprd](./vscode-debugging-daprd)
+
+## SDKs
+
+* [Serialize state and transient objects](./serialize)
```

Comment:
I don't think we should use the term "transient object", it sounds odd.  Maybe "method invocation data/objects"?

Reply:
Changed it.